git/helpers.sh

```diff
@@ -1,3 +1,15 @@
+#
+# Open file or directory in web browser; defaults to current directory and branch
+# Syntax: git-web [$file_or_dir] [$branch]
+#
+function gitweb() {
+  file=${1:-""}
```

Comment:
The common convention is to **use `lower_case` names for variables that aren't exported to the shell**. Sources:
1. https://stackoverflow.com/a/673940/4472510
2. https://unix.stackexchange.com/a/42849/112657
3. https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap08.html